Doc/library/tkinter.messagebox.rst

```diff
@@ -21,19 +22,169 @@ limited to:
 
    Create a default information message box.
 
+   The following options are supported:
+
+      *command*
+         Specifies the function to invoke when the user closes the dialog.
+         The name of the button clicked by the user to close the dialog is
+         passed as argument.
+         This is only available on macOS.
+
+      *default*
+         Gives the :ref:`symbolic name <messagebox-buttons>` of the default button
+         for this message window (:data:`OK`, :data:`CANCEL`, and so on).
+         If this option is not specified, the first button in the dialog will
+         be made the default.
+
+      *detail*
+         Specifies an auxiliary message to the main message given by the
+         *message* option.
+         The message detail will be presented beneath the main message and,
+         where supported by the OS, in a less emphasized font than the main
+         message.
+
+      *icon*
+         Specifies an :ref:`icon <messagebox-icons>` to display.
+         If this option is not specified, then the :data:`INFO` icon will be
+         displayed.
+
+      *message*
+         Specifies the message to display in this message box.
+         The default value is an empty string.
+
+      *parent*
+         Makes the specified window the logical parent of the message box.
+         The message box is displayed on top of its parent window.
+
+      *title*
+         Specifies a string to display as the title of the message box.
+         This option is ignored on macOS, where platform guidelines forbid
+         the use of a title on this kind of dialog.
+
+      *type*
+         Arranges for a :ref:`predefined set of buttons <messagebox-types>`
+         to be displayed.
+
+
+   Returns the symbolic name of the selected button.
```

Comment:
```suggestion
   Returns the symbolic name of the selected button, which is the one clicked on or the one highlighted when the Enter key is hit.  Change the button selected with the Tab key or arrow keys.
```
Could have :key: markup, but I forget the details.  I am not sure what the truth is for the second sentence.  Same on all systems?  Others will not know either ;-).

Reply:
It adds too much details which are not even correct. You can select the button by pressing Space or a shortcut (Alt+the underscored letter). Arrow keys do not change the default button on X Window. Some other keys (e.g Shift-Tab) can change the default button.

Tk uses default behavior on every platform, and it is not a good place to describe it.